src/routes/wohnen/components/OfferListItem.js

```diff
@@ -1,21 +1,54 @@
 // @flow
 
 import React from 'react'
-import style from './OfferListItem.css'
-import WohnenOfferModel from '../../../modules/endpoint/models/WohnenOfferModel'
+import WohnenOfferModel from 'modules/endpoint/models/WohnenOfferModel'
+import WohnenFormData from 'modules/endpoint/models/WohnenFormData'
+import styled from 'styled-components'
+import ListElement from 'modules/common/components/ListElement'
 
 type PropsType = {
-  offer: WohnenOfferModel
+  offer: WohnenOfferModel<*>
 }
 
+const Description = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-left: 10px;
+  padding-bottom: 5px;
+`
+
+const Title = styled.div`
+  padding: 15px 5px 5px;
+  font-weight: 700;
+`
+
+const Price = styled.div`
+  align-self: flex-end;
+`
+
 class OfferListItem extends React.Component<PropsType> {
   render () {
     const offer = this.props.offer
-    // todo: display offers
-    // return <a href={offer.url} className={style.job} target='_blank' >
-    return <div className={style.title}>{offer.email}</div>
-    // <div className={style.description}>{offer.location}</div>
-    // </a>
+
+    if (offer.formDataType === WohnenFormData) {
+      const specificOffer: WohnenOfferModel<WohnenFormData> = offer
+      const accommodation = specificOffer.formData.accommodation
+      const costs = specificOffer.formData.costs
+      const landlord = offer.formData.landlord
+
+      return <ListElement>
+        <Title>{landlord.firstName} {landlord.lastName}</Title>
+        <Description>
+          <div>
+            <div>{accommodation.totalArea} m²</div>
```

Comment:
We use `qm` as unit in all our other platforms, so maybe we should stay consistent here...

Reply:
*Reminder*